worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -242,20 +248,19 @@ public String getDisplayName() {
     //FAWE start
     private final Relighter relighter;
     private final boolean wnaMode;
-
     @Nullable
     private final Region[] allowedRegions;
+    private @Nullable List<TracingExtent> tracingExtents = null;
 
     @Deprecated
```

Comment:
If we're at it, javadocs with proper note what to be used and marked for removal are quite handy.